workloads/kube-burner/common.sh

```diff
@@ -136,3 +153,19 @@ cleanup() {
   oc delete ns -l kube-burner-uuid=${UUID}
 }
 
+get_pprof_secrets() {
+ export certkey=`oc get secret -n openshift-etcd | grep "etcd-serving-ip" | head -1 | awk '{print $1}'`
+ echo `oc extract -n openshift-etcd secret/$certkey`
```

Comment:
Why the leading echo?

Reply:
echo makes sure that the cert and key are displayed on the same line , if only oc extract .. then it displays on 2 lines.